container/service_integration_test.go

```diff
@@ -54,38 +54,38 @@ func TestIntegrationStopNotExistingService(t *testing.T) {
 	require.Nil(t, err)
 	namespace := []string{"TestStopNotExistingService"}
 	err = c.StopService(namespace)
-	require.Nil(t, err)
+	require.NotNil(t, err)
```

Comment:
I will do it in another PR. There is many places to replace `require.Nil(t, err)` and `require.NotNil(t, err)` by `require.NoError(t, err)` and `require.Error(t, err)`.
Also, one test file is still using `stvp/assert`.

Reply:
Sure, I didn't want to do everything at once. Just to make sure new assertions have more meaningful errors messages. But it could be one big simple PR as well

Reply:
why this test changed in the first place ? The stop should not return an error if the service doesn't exists. If it doesn't exists it means that it's not running. Agree that we should also have a kind of NotFound error just want to make sure that this change in the test really needs to be done

Reply:
I will investigate but I think the function `StopService` is not behaving the same way since it wait for actual deletion of Docker's ressources

Reply:
I confirm the function is behaving differently since this commit https://github.com/mesg-foundation/core/commit/0366e060934827a179e417c0b6a25170ebc6b246

Reply:
Fix in new PR: https://github.com/mesg-foundation/core/pull/505